### PR TITLE
issue81 upgrade smart contract to cosmwasm provwasm 1.0.0 beta

### PR DESCRIPTION
**Pull request by leeduan**
Upgrade to 1.0.0-beta of cosmwasm/provwasm

### CODE REVIEW

File: smart-contract/schema/init_msg.json [hunk 1]
```diff
@@ -9,16 +9,6 @@
     "vote_duration"
   ],
   "properties": {
-    "admin_weight": {
```
Comment (jtalis) [inline smart-contract/schema/init_msg.json:12]: are these schema changes unrelated to 1.8 upgrade?

**Review by jtalis — APPROVED** [review]

File: smart-contract/schema/init_msg.json (hunk 1 above)
Comment (leeduan) [reply smart-contract/schema/init_msg.json:12]: these changes were from before but i'm not sure why the json werent updated...

Comment (leeduan) [reply smart-contract/schema/init_msg.json:12]: https://github.com/provenance-io/digital-currency-consortium/pull/75/files

**Comment by leeduan** [comment]
tested the endpoints against the old instance of localnet smart contract from before upgrading and everything seems to work fine

File: smart-contract/README.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ Grant for bank 1
 
 ```bash
 provenanced tx marker grant-authz \
-    tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
+    tp14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9s96lrg8 \
```
Comment (leeduan) [inline smart-contract/README.md:309]: 1.8 seems to have changed the length of the smart contract addresses on instantiation

File: service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/BankService.kt [hunk 3]
```diff
@@ -62,6 +62,7 @@ class BankService(
 
             val registration = AddressRegistrationRecord.findByBankAccountUuid(bankAccountUuid)
             checkNotNull(registration) { "No registration found for bank account $bankAccountUuid for coin mint $uuid" }
+            check(registration.isActive()) { "Registration for bank account $bankAccountUuid is not active" }
```
Comment (leeduan) [inline service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/BankService.kt:65]: squeezed this in here from testing

**Pull request merged**